lib/nfc/parsers/myki.c

```diff
@@ -0,0 +1,104 @@
+#include "nfc_supported_card.h"
+
+#include <nfc_worker_i.h>
+
+#include "furi_hal.h"
+
+bool myki_parser_verify(NfcWorker* nfc_worker, FuriHalNfcTxRxContext* tx_rx) {
+    furi_assert(nfc_worker);
+    MifareDesfireData* data = &nfc_worker->dev_data->mf_df_data;
+
+    if(!furi_hal_nfc_detect(&nfc_worker->dev_data->nfc_data, 300)) return false;
```

Comment:
I'm not quite sure why this extra operation is needed [compared to ultralight](https://github.com/flipperdevices/flipperzero-firmware/blob/a0814aab31d7f08eeffa5d424769ab1e303f4eb3/lib/nfc/parsers/all_in_one.c#L47), but the card fails to read successfully without it. Would anyone be able to shine some light on this?